src/material/input/testing/native-option-harness.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {ComponentHarness, HarnessPredicate} from '@angular/cdk/testing';
+import {NativeOptionHarnessFilters} from './native-select-harness-filters';
+
+/** Harness for interacting with a native `option` in tests. */
+export class MatNativeOptionHarness extends ComponentHarness {
```

Comment:
I went with the "native" naming here to avoid confusion with `MatSelect`, but I'm open to other suggestions on what to call it.